packages/react/src/schematics/library/library.ts

```diff
@@ -45,6 +45,7 @@ import {
 } from '../../utils/versions';
 import { assertValidStyle } from '../../utils/assertion';
 import { extraEslintDependencies, reactEslintJson } from '../../utils/lint';
+import { toJS } from '@nrwl/workspace/src/utils/rules/to-js';
```

Comment:
**I'm missing 2 things (for both react app/lib):**

- adding `**/*.js` to `tsconfig.includes`
- adding `"allowJs": true` to tsconfig (or this library/app will be transpiled only via babel ?)

or am I missing something ?

**Question:**

What about jest tests ? they'll be still executed via ts-jest ? (although not necessary -> https://github.com/nrwl/nx/issues/1720)

thx!

Reply:
1. Yes, the tsconfig should have `*.js` and `*.jsx` files--I'll update PR.
2. `allowJs` is already set to `true` for react apps and libs. Transpilation via babel ignores this setting, but it is used by the type checker (and any IDE integration).
3. We can tackle `babel-loader` for jest separately in the issue you mentioned.

Thanks!

Reply:
awesomeness ! thanks a lot